<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8"/>
	<title></title>
	<meta name="generator" content="LibreOffice 6.0.7.3 (Linux)"/>
	<meta name="author" content="Czipa Erik"/>
	<meta name="created" content="2019-05-08T10:22:00"/>
	<meta name="changedby" content="Microsoft Office User"/>
	<meta name="changed" content="2019-05-08T10:22:00"/>
	<meta name="AppVersion" content="16.0000"/>
	<meta name="Company" content="HP"/>
	<meta name="DocSecurity" content="0"/>
	<meta name="HyperlinksChanged" content="false"/>
	<meta name="LinksUpToDate" content="false"/>
	<meta name="ScaleCrop" content="false"/>
	<meta name="ShareDoc" content="false"/>
	<style type="text/css">
		@page { size: 8.27in 11.69in; margin: 0.98in }
		p { margin-bottom: 0.1in; direction: ltr; color: #00000a; line-height: 115%; text-align: left; orphans: 2; widows: 2 }
		p.western { font-family: "Times New Roman", serif }
		p.cjk { font-family: "Calibri"; so-language: hu-HU }
		p.ctl { font-family: "Times New Roman" }
	</style>
	<link href="style.css" rel="stylesheet" type="text/css">
</head>
<body lang="hu-HU" text="#00000a" dir="ltr">
<p class="western" style="margin-bottom: 0in; line-height: 100%"><a name="_GoBack"></a>
<font face="Arial, serif"><font size="5" style="font-size: 20pt"><span lang="en-US"><b>Help
page</b></span></font></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><font size="4" style="font-size: 14pt"><span lang="en-US"><b>Motif
view</b></span></font></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">Short
description: Find the average/median protein positional and occupancy
frequency information on a set of given transcription factor binding
sites. </span></font>
</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">The</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">motif</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">view</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">shows</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">average</span></font><font face="Arial, serif"><span lang="en-US">/</span></font><font face="Arial, serif"><span lang="en-US">median</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">positions</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">occupying</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">proteins</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">on</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">instances</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">a</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">given</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">transcription</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">factor</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">motif (</span></font><font face="Arial, serif"><span lang="en-US">on
the whole or a portion of the consensus motif set</span></font><font face="Arial, serif"><span lang="en-US">)</span></font><font face="Arial, serif"><span lang="en-US">.
The reference point is marked </span></font><font face="Arial, serif"><span lang="en-US">as</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">zero</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">and</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">it</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">represents</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">center</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
the </span></font><font face="Arial, serif"><span lang="en-US">motif</span></font><font face="Arial, serif"><span lang="en-US">.
</span></font><font face="Arial, serif"><span lang="en-US">The</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">scatter</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">plots</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">provide</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">information</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">about</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">frequency</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">co</span></font><font face="Arial, serif"><span lang="en-US">-</span></font><font face="Arial, serif"><span lang="en-US">occupancies</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">and</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">distance</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">distribution</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">standard</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">deviations</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">distinct</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">proteins</span></font><font face="Arial, serif"><span lang="en-US">.
</span></font><font face="Arial, serif"><span lang="en-US">The</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">latter</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">can</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">correlate</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">with</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">physical</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">distance</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">between</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">protein</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">and</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">the</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">DNA</span></font><font face="Arial, serif"><span lang="en-US">
(</span></font><font face="Arial, serif"><span lang="en-US">direct</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">or</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">indirect</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">binding</span></font><font face="Arial, serif"><span lang="en-US">).
</span></font><font face="Arial, serif"><span lang="en-US">Every</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">dot</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">represents</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">a</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">ChIP</span></font><font face="Arial, serif"><span lang="en-US">-</span></font><font face="Arial, serif"><span lang="en-US">seq</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">experiment,</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">which</span></font><font face="Arial, serif"><span lang="en-US">
is </span></font><font face="Arial, serif"><span lang="en-US">colored</span></font><font face="Arial, serif"><span lang="en-US">
according to the </span></font><font face="Arial, serif"><span lang="en-US">type</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">target</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">protein</span></font><font face="Arial, serif"><span lang="en-US">.
</span></font><font face="Arial, serif"><span lang="en-US">The</span></font><font face="Arial, serif"><span lang="en-US">
displayed dots </span></font><font face="Arial, serif"><span lang="en-US">can</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">be</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">filtered</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">based</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">on</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">cell</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">type</span></font><font face="Arial, serif"><span lang="en-US">/</span></font><font face="Arial, serif"><span lang="en-US">cell</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">line</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">or</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">target</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">proteins</span></font><font face="Arial, serif"><span lang="en-US">.
</span></font>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<img src="help_html_6df2f40d95d3a84.gif" name="Kép 52" align="left" hspace="11" vspace="1" width="390" height="539" border="0"/>
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">The
motif view appears as an interactive scatterplot, which gives
information about the given JASPAR core motif instances in the genome
(global analysis of consensus motif set, see above), the overlap
frequency between motifs, and ChIP-seq peaks and the average/median
positioning of summits from different experiments. Every dot on the
chart belongs to a single ChIP-seq experiment. The dots are placed
depending on the relation between the positioning information and the
adjusted motif center. (The position weight matrix of the adjusted
motif is shown in the bottom-right corner. The center of the motif is
marked with “0” on the scale below.) The motif of interest can be
set in the “Set a motif” dropdown box. In the write boxes, you
can modify what data is displayed and the minimum and maximum values
of the y axis (standard deviation or element number, this will be
discussed later in this description). Important: All of the changes
will only be displayed after clicking on the “Resend Data” button
below. </span></font>
</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">If
the user hovers the cursor on a given dot, a tool-tip will appear,
which gives information about the ChIP-seq experiment, including the
name of the experiment, cell type, target protein, and quantified
information about summit positions (average/median distance, standard
deviation of distances, and overlap number). The dots are </span></font><font face="Arial, serif"><span lang="en-US">colored</span></font><font face="Arial, serif"><span lang="en-US">
according to the </span></font><font face="Arial, serif"><span lang="en-US">type</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">of</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">target</span></font><font face="Arial, serif"><span lang="en-US">
</span></font><font face="Arial, serif"><span lang="en-US">protein</span></font><font face="Arial, serif"><span lang="en-US">.
The legend with color codes is visible on the right-hand side of the
chart, which is also interactive. Clicking on a specific target
protein name in the legend section can hide the respective dots from
the chart. The large amount of displayed dots can be overwhelming in
the data review, so we created a “Clear all dots out” button to
hide all of the points of the chart. The specific spots can be called
back one-by-one by clicking on the factor name in the legend. Using
this process, we can compare the positioning of proteins of interest.
 All of these steps are revocable by clicking on the “Show all
dots” button. </span></font>
</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"> <font face="Arial, serif"><span lang="en-US">The
X axis is constant and represents the distance from the center of the
adjusted JASPAR CORE motif (the distance is measured in base pairs).
The Y axis is adjustable, you can choose to display the number of
summit-motif overlaps or the standard deviation of summit positions. </span></font>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">As
was previously mentioned, we created consensus motif sets for the
JASPAR core motifs, which represent all of the possible binding sites
for different factors (documentation link). The power of the ChIP-seq
technique resides in its comparability, which means in our case, that
we can easily investigate the relationship between motif centers and
nearby summit positions. We define this in distances (measured in
base pairs) then visualize the average, frequency, and standard
deviation on scatterplots. Several motifs show a high density of
overlapping summits. Binding sites like CTCF or AR motifs are popular
among transcription factors/ co-factors in the case of genomic
co-localization. Genomic regions like super enhancers are
collectively bound by an array of factors. At the core of these
regions are transcription factor motif(s), which is/are covered with
a collection of different proteins. Since we investigate whole sets
of possible transcription factor binding sites and we visualize the
average values of all possible co-localizing factors (the summit
positions of these factors), we can run into some rather crowded
scatterplots. To resolve this problem, we can use the previously
mentioned “Clear all dots out” function or we can utilize the
factor unification solution. The latter is based on consolidating
different ChIP-seq experiments (even from different cell types),
which have a common antibody target. The unification step averages
the summit positions of the affected experiments (average of
averages), the standard deviations (average of standard deviations),
and the element numbers. To activate this display you can choose
between two buttons: “average standard deviation vs. average of
average positions” or “average element numbers vs. average of
average positions”. The two features differ in the values of the y
axis. The first one displays the average of standard deviations, the
second shows the average number of motif-summit overlaps per
experiment.  </span></font>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US"><b>Interoperability
between the different views</b></span></font></p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">The
motif view provides a global picture about the transcription factor
binding sites and their occupying ChIP-seq signals. Thus, the motif
view is a genomic bird’s-eye view, which is a useful tool for
identifying intriguing phenomena. However, in order to properly
understand what we are seeing, we need to be able to take a closer
look. </span></font>
</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">As
mentioned in the Help section, the diagram was made to be
interactive. The dots on the diagram (which all represent a specific
ChIP-seq experiment) can be marked by clicking on them. The
attributes (cell line, antibody, SRA ID, element number) of the
selected experiments (a maximum of 3 experiments can be selected at
the same time) appear below the chart. If you want to investigate the
appointed highlighted data, you can choose from the following
options: </span></font>
</p>
<ul>
	<li/>
<p style="margin-bottom: 0in; line-height: 100%"><font face="Calibri, serif"><font face="Arial, serif"><span lang="en-US">If
	you click on the “to pair shift view” button, it will navigate
	you to the distance distribution chart of the summit positions (of
	the selected experiments) compared to the adjusted motif (the same
	motif, which was displayed in the motif view). </span></font></font>
	</p>
	<li/>
<p style="margin-bottom: 0in; line-height: 100%"><font face="Calibri, serif"><font face="Arial, serif"><span lang="en-US">To
	browse the genomic locations of peak-motif co-occurrences in the
	genome browser, click on the “to the jbrowse” button.</span></font></font></p>
	<li/>
<p style="margin-bottom: 0in; line-height: 100%"><font face="Calibri, serif"><font face="Arial, serif"><span lang="en-US">You
	can check the frequency of co-occupancy between the selected factors
	by clicking on the “to Venn diagram” button. This will open a
	classical logic diagram display mode, where the motif related
	co-appearance of summits can be seen.</span></font></font></p>
</ul>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%; page-break-before: always">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><font size="4" style="font-size: 14pt"><span lang="en-US"><b>Pair
shift view</b></span></font></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">T<img src="help_html_47772c16f749187f.gif" name="Kép 53" align="left" hspace="11" width="383" height="540" border="0"/>
he
motif view displays the statistical data (occurrence frequency,
average/median distance related to the motif, and distance standard
deviation) of all the ChIP-seq experiments, which contain overlapping
peaks with any instances of an adjusted motif type (e.g.: all CTCF
motif). The positioning of the different factors related to the motif
has a central role in this view but this view’s resolution is too
low to see the details of position distribution. </span></font>
</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">The
pair shift view shows the summit distance distributions of the
selected ChIP-seq data (a maximum of 3) related to a motif as a
histogram. The X axis represents the distance (measured in base
pairs) from the middle of the given motif, which is marked as the “0”
point. The numbering of the axis is consistent with the position
weight matrix below the diagram.  The Y axis shows the frequency of
summit occurrences at the relative position (at a given base pair)
relative to the motif center. In the case of well-defined protein
topology with high overlap frequency and close DNA localization, the
curve has a bell-like pattern (normal distribution-like). According
to our observations, the narrowness of the curve is inversely
proportional to the protein’s physical distance from the DNA
(direct or indirect binding). This relationship can be detected when
looking at the standard deviations as well (motif view). Factors with
low overlap frequency and no position preference show plateau
distribution. </span></font>
</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><a name="__DdeLink__1528_1325945844"></a>
<font face="Arial, serif"><span lang="en-US">To visualize the data,
we need to set the options. We recommend that you set the motif first
in the dropdown box below “Set a motif”. After you choose a motif
of interest, you can set which experiments you want to investigate.
(You can select them using the dropdown menus, or you can navigate to
pair shift view comparison from motif view after you highlighted the
dots (experimental data) that you are interested in. You can read
about this in ”Details”, found in the Motif view Help section. To
select the ChIP-seq data of interest, set the attributes of the
experiment in the dropdown boxes from right to left: in the first box
select the cell type, in the second you can pick the antibody, and in
the third box you can choose by the name of the experiment. The
experiment name is related to the experimental attributes: the tissue
type and the origin of the cell type, the target protein name, and
the SRA experiment ID. When you set all the parameters, click on the
“Resend Data” button. Following the page refresh, the updated
data will be displayed. The minimum and maximum values of the axes
are configurable as well in the text boxes below the diagram. A
rolling mean with a 5 bp frame was applied to smooth the frequency
curves.</span></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><font size="4" style="font-size: 14pt"><span lang="en-US"><b>VennView</b></span></font></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">The
diagrams of the motif view cumulatively represent the statistical
data of all occupying ChIP-seq experiments (occurrence frequency,
average/median distance related to the motif, and distance standard
deviation) on all instances (consensus motif set) of an adjusted
motif type. The co-occurrence frequency of distinct ChIP-seq summits
from different experiments is not taken into account here. To fill
this gap, we created a Venn diagram view. The Venn diagram displays
all possible logical relations between a collection of different
sets. In our case, the sets are the motifs that overlap with the
peaks of a chosen ChIP-seq experiment and the relation is the number
of common motifs that are simultaneously occupied by these
experiments.  </span></font>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><font size="4" style="font-size: 14pt"><span lang="en-US"><b>Experiment
view</b></span></font></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><font face="Arial, serif"><span lang="en-US">At
the early stage of our work, we collected 4068 human ChIP-seq data
from public databases (NCBI SRA, ENCODE). 3782 experiments were
successfully processed and used in the following steps of the
analysis. The basic information of this data is at least as crucial
as the final results. As previously mentioned, we tried to use a wide
variety of ChIP-seq data considering both the origins (cell type,
tissue) and the target proteins. To track the source of the data, we
created an “Experiment view”, which is a more manageable and
readable way to browse essential information about the distinct
experiments by putting all of the data into a simple table. The
search interface of this view is quite similar to the Pair shift view
and Venn diagram view.</span></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0in; line-height: 100%">
<br/>

</p>
<p class="western" style="margin-bottom: 0in; line-height: 100%"><br/>

</p>
</body>
</html>
